### ISSUE TITLE
undefined method 'update'

### ISSUE THREAD

**Issue by hansy** [amberframework/granite#252]
Granite v0.12.1

How do I update a record?

```
field = Field.find 1
field.update(title: "Some title") if field
```

Error is `undefined method 'update' for Field (did you mean '__update'?)`.

**Comment by Blacksmoke16** [comment]
Can you share your model class?

**Comment by hansy** [comment]
Oh yeah absolutely.

```
class Field < Granite::Base
  adapter pg
  table_name fields

  id : Int64
  field title : String
  timestamps
end
```

**Comment by Blacksmoke16** [comment]
I wonder if your class name is conflicting with a Granite one.  Try renaming it like `Foo` or something.

**Comment by c910335** [comment]
`#update` is a new feature in master branch, not yet available in v0.12.1
